cli/azd/internal/repository/infra_confirm.go

```diff
@@ -103,10 +102,45 @@ func (i *Initializer) infraSpecFromDetect(
 		if svc.Docker == nil || svc.Docker.Path == "" {
 			// default builder always specifies port 80
 			serviceSpec.Port = 80
-
 			if svc.Language == appdetect.Java {
 				serviceSpec.Port = 8080
 			}
+		} else {
+			ports := svc.Docker.Ports
```

Comment:
Since we're also doing the prompting here -- let's think about moving the logic from L179-L203 here.

Reply:
I think we can apply consistency to the messages and make it simpler.

Currently we have:
![image](https://github.com/user-attachments/assets/32119fc3-64f3-47c5-ad31-cf65e7a048b9)

This should look more like: 

```bash
? What port does 'dotnet' listen on?
3100
3233
Other
```

I think it's okay to omit how we did the detection for now. Perhaps in the future if we do have user asks, we can decide how to best present this information.

Reply:
Updated.